docker-deps/artifactory_key.pub

```diff
@@ -0,0 +1,31 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v1
+
```

Comment:
I want to understand where and how did you get this key. Can there be any issue with permissions when others use this key to run things n their machine?

Reply:
This is the public key for the artifactory repositories. Installing packages with apt will verify the authenticity of the repository by checking the public key of the repository.